### ISSUE TITLE
Suggestion: Command Crew

### ISSUE THREAD

**Issue by dplepage** [endless-sky/endless-sky#513]
Fleets would be slightly more balanced if you had to pay crew members extra to captain your ships. One way to do this would be to have every ship require some number of crew, some number of officers, and one captain, with higher pay for officers and significantly higher pay for captains. You'd fill the "captain" slot on your own ship, but every additional ship in your fleet would require much higher pay for the captain, and for large ships like Bactrians you'd also be paying extra for some of your crew because they'd have to be officers in charge of the rest of the crew.

**Comment by Nightmareman** [comment]
The balance of fleets is pretty close to being fine the way it is once you get a fleet where you pay 400k every day you will understand. The missions become pointless and only way to fund the fleet is ether with capturing enemies or doing the lucrative trade routes. The smart thing to do if you are doing anything but those 2 activities is to park 90% of your fleet or you will be hemorrhaging money quicker then a cut artery.

**Comment by dplepage** [comment]
By the time you're paying 400K for your fleet you've already won the game.

Certainly in EV the game became pretty pointless after you'd finished all the mission chains - I always felt that the ability to demand tribute from planets was a clunky feature that Ambrosia bolted on just to keep the game from being entirely boring after the missions.

I'm mainly concerned with the middle of the mission chains, where you can breeze through just by farming warships until you have a fleet capable of winning all the missions easily. I also think command crew would add another layer of realism to the game - why shouldn't a crewperson in charge of an entire ship ask for more money than a lowly deckhand on your own frigate?

But maybe this doesn't bother anyone else.

**Comment by Nightmareman** [comment]
I would agree to command crew only if they are optional only and give bonuses to the ships operation based on dialog in the game there is a shortage of work which is probably why you can get away with paying everyone the same. This of course would change if they brought unique and specialized skills to your ships in which case they are obviously more valuable but in a place where the number of jobs is short you could get away with putting someone who is competent yet not super skilled in each job for much cheaper then if the reverse were true.

**Comment by bobafetthotmail** [comment]
I like a lot the idea, although it's basically another money sink. 
Here the main issue is that there is little to do since missions don't scale. 
Increasing the costs of keeping a fleet don't solve the problem and only frustrates players, for now.

After that is solved I would like to see captain costs too in my fleet costs.

>"The smart thing to do if you are doing anything but those 2 activities is"

...to dominate planets to demand tribute. Once enough planets pay you, you can offset the costs.

>" This of course would change if they brought unique and specialized skills to your ships in which case they are obviously more valuable"

A captain (more like team of deck officers) lets a ship move and fight without the player's direct manual control, does that count as "unique and specialized skill" enough to justify a wage increase?

**Comment by Nightmareman** [comment]
No its not a unique and specialized skill as most people with average intelligence can give the commands necessary to run a ship in the limited fashion they do in the game. Not sure if you have seen the types of people in real life who can run management in a semi-competent way but it don't require much training to do the bare minimum which is why I suggested it be optional same price for those who are only competent in the position but a higher price for those who are actually highly skilled in the position and as such offer bonuses for the efficiency in how the ship runs. Its the same for the military in real life have seen more then enough people whose intelligence, command and management skills are only what I would consider average get promoted to Major or even up to Colonel. The way the job system and military works is if you do a good job at something you get promoted eventually and it keeps happening over and over till the person ends up in a position that they do poorly in then they stop climbing the ladder but the whole system ends up suffering because the person is now in a spot where they just can't handle the load. There are some exceptions people who know their own limits and refuse promotion and hats off to those people but most people will accept the promotion without even thinking about how much different the new job will be.

So to summarize everything I just said same price for those who are only competent yet a higher wage for those who are very skilled at the assigned position for whatever would be considered "deck officers". I would like to see new unique and random made NPCs added to the game so this would actually be a good function to add as it would give a reason to add those NPCs the unique NPCs could even come with their own mini-story line missions kind of like the companions of other RPGs offer. Would give a good extra layer of depth to the game and I am always for an extra layer of story depth.

**Comment by bobafetthotmail** [comment]
>"No its not a unique and specialized skill as most people with average intelligence can give the commands necessary to run a ship in the limited fashion they do in the game."

Ah come on... to drive the ship like it is driven in-game I can write a bunch of code myself and not pay any crew at all... just like the game AI is a piece of software and not a person.

>"Not sure if you have seen the types of people in real life who can run management"

Not sure if you have ever seen a manager (good or bad) that is payed like a normal worker, outside communist utopias, that is.

**Comment by Nightmareman** [comment]
@bobafetthotmail Combine it with my comment before it and you would understand perfectly in the other arguments you have shown the same flaw not paying attention to the conversation overall or reading all of a persons comments or the whole comment.

But regardless I will summarize the whole thing for you. The in game lore shows a shortage of steady jobs and since just being competent in management and command isn't a specialized skill something an average can do it would be logical you could get away with paying them the same as everyone else. Especially since they don't have to actually make any big independent decisions those are all left to the player and all they have to do is follow the player around and follow rather simplistic orders. Now as I said in the previous comment someone who is actually very good at the officer position would be payed more but they would also give bonuses to ship operation as they are something far better then average.

**Comment by bobafetthotmail** [comment]
You are right in saying that something like this is just another cost on top of other costs and that it does not make the game better.

But please don't use total non-sense arguments like "a captain of a ship or a leader is something that everyone can do so I can pay him the same", or "the in-game AI is dumb so I don't see why I should pay people to do that".

**Comment by Nightmareman** [comment]
@bobafetthotmail Average person can do it competently not everyone can do it two totally different meanings and it is a valid argument. Do you even comprehend what I write or do you just scan through and come to a random conclusion without understanding the whole context of the comment.

**Comment by bobafetthotmail** [comment]
it isn't a valid argument because the game AI (the actual driver of the ships) is dumb, so whatever is the in-game activity you are looking at, any human being can do it better.

**Comment by Nightmareman** [comment]
@bobafetthotmail Again you prove you aren't reading the whole comment or just not comprehending it I gave you real life examples that show that running something at a competent level doesn't require any special level of intelligence or hard to learn skills its something an average person can do and its a fairly easy to learn skill set. Here going to give you a perfectly logical reason on why they aren't going to get payed more for being only competent. Its going to be even easier on the ships crews as its the year 3000 so pretty sure there is a fairly high level of automation and computer assistance. The escort ships version of a bridge crew is likely far simpler then what is required of a current day bridge crew as 1) The escorts don't have to make any big independent decisions as such you don't need anyone with special qualifications they only need to be competent at each role. All big decisions are left to the player you decide destinations, enemy targets, cargo, the management of personal, management of equipment all the major stuff you take care of all they have to do is follow. 2) I will use the pilot as an example only needs to be competent enough to not run into anything, point the ship in the right direction and landing with the level of computer assistance the year 3000 is going to have 99% of the fine details probably aren't even done by him he likely just enters commands. 3)See when you use a bit of logic it all makes sense its going to be the same for the rest of the bridge crew they are going to be relaying commands and doing the few small decisions necessary that are too small for the player character and that requires the human touch rather then the computer. Rest of the crew will be doing the menial but necessary day to day tasks of a ship. They each have their own role but because of the level of automation 3000 would bring each job isn't really much harder then the other and no requirement for high amount of training just need enough to get by which is what the basic crew man has. The only one who really needs to truly have a grasp of what he is doing and whats going on is you the player character as you are the real Captain all the big decisions rest on you as you are the only one that has to look at the big picture everyone else gets to focus on their individual job as your underlings which is assisted by the computer.

**Comment by bobafetthotmail** [comment]
The reason a leader figure (anywhere) is payed more is because the leader's job is to make sure that everyone under his command is doing their job well (and I should not need to explain why this is needed), and also to put down the little smartypants that think they know better.
It is something that not everyone can do decently, and it is (much) more tiring than standard worker duties.

It is a task that is likely done by the player on his own ship, but is vacant in the others of the fleet.

>"level of computer assistance the year 3000"

The level of automation and AI will outright negate the need for a crew at all well before 2200 AD, go figure a millenia from now.

In case you did not notice, the crew onboard is just a convenient sticker over some game mechanics.
"daily money sink"
"capturing balancing system"
"balance between passenger space and capturing capacity"

In case the crew is removed, these game mechanics will need a different name, but will have to remain.

But since we have crew, a leadership figure will make sense.

**Comment by Nightmareman** [comment]
The players character is a leadership figure. Its just as the old man said at the very start you are the one that manages crew and keeps track of the whole fleet all of that is on you. So all anyone else on the other ships has to do is relay your commands and make small decisions something an average person can do and in a galaxy where jobs are short it is easy to get away with paying them the same as all the others if they aren't anything special. The player and his crew aren't military or even militia so no real ranks it is the player at the top and all the crew under him. Use some logic and it would make sense but you are too busy thinking about the way things currently are or how they would work in a perfect world. In a galaxy where jobs are short and poverty common finding someone to fill any necessary positions would be easy as it is a steady job. Think they are going to risk doing something to upset the player and get dropped off at the first habitable planet, when there are millions lining up for that same job? If the person is only average there will be no problems but if they truly excel then they would get payed more but they would also give bonuses of some sort like I suggested in my second comment in this thread.

**Comment by bobafetthotmail** [comment]
I repeat the important part of my post:

The leader's job here is to make sure that everyone under his command is doing their job well (and I should not need to explain why this is needed).

**Comment by Nightmareman** [comment]
The players character is the leader so your point is invalid. Also as I stated earlier its not as special of a skill as you think it is an average can be trained to do it competently I gave you tons of real life examples you didn't read them or you just don't comprehend. If you had of experienced this type of thing in the real world you would know it is pretty common for people who really aren't special in any way to get management and leader positions. I've seen tons and tons of people promoted to such positions and are bad at it or only competent at it over people who would probably excel at the same job. Its part of life, life isn't fair and that will never change. If you would use logical thinking combined with real life experience you would know this to be true but instead you just spout what you believe rather then what actually happens.

**Comment by dplepage** [comment]
Y'all realize that this is game, right?

You don't need to sit here arguing about whether "real world" spaceship commanders would need to be paid more, because @endless-sky can set up the universe any way they want.

E.g. any of the following could be part of the game's mythos:
 * Running a ship is hard, because of details that we don't get to see (the same way we don't get details of how the crew get fed or who manages the janitors on your spaceships) so you have to pay commanders more than normal crew, OR
 * Ship AIs are sophisticated enough in the future that the commanders don't actually have to do very much, which is why you don't pay them any more than normal crew, OR
 * Running a ship is hard, and commanders should get paid more, but we don't include that because we don't want that complexity and this is a game, not a reality simulator.

The question you SHOULD be debating is: Does this actually add anything to the experience of the game, or is it just a needless bit of complexity?

Personally, I like this because it might keep crew salaries a little more relevant - in the early game, you care a lot about crew salaries, but once you have millions of credits it quickly becomes something you don't ever think about anymore. That strikes me as odd, because it means it's an extra layer of complexity that you *only* care about when you're a fairly new player, which is usually not the direction you want complexity to go in. Having to think about the extra costs of commanders is a level of complexity that won't show up until you've been playing long enough to start buying warships or building a small fleet, so it's complexity in the right direction - things get more complicated as you get better at the game.

But I also prefer flying solo to managing a large fleet, and this change would definitely make it more expensive to maintain a large fleet. From a lot of other discussion it sounds like most players really like the fleet management stage of the game, so a change that delays that stage is probably a bad idea.

So unless anyone besides me wants to speak up in favor of this change, I'll go ahead and close this request.

**Comment by Nightmareman** [comment]
The inverse is true about what you stated about crew salaries I pay far more attention to them late game then I did early game just because I am paying half a million per day to them so I have to make sure I am always doing something that makes money otherwise my bank account starts taking serious hits. Which is why I park the majority of my fleet if I am doing story line missions or something else other then normal bulk trading.

**Comment by bobafetthotmail** [comment]
>"The question you SHOULD be debating is: Does this actually add anything to the experience of the game, or is it just a needless bit of complexity?"

Well, my first post in the issue thread is exactly about this.
I said:

I like a lot the idea, although it's basically another money sink.
Here the main issue is that there is little to do since missions don't scale.
Increasing the costs of keeping a fleet don't solve the problem and only frustrates players, for now.

After that is solved I would like to see captain costs too in my fleet costs.

>"Having to think about the extra costs of commanders is a level of complexity that won't show up until you've been playing long enough to start buying warships or building a small fleet, so it's complexity in the right direction - things get more complicated as you get better at the game."

I agree with this, but before you add more money sinks, you need to add stuff to do.

**Comment by endless-sky** [comment]
Salaries are already confusing enough for new players without making some salaries cost more than others. And, by the time a player has a large fleet, they aren't worrying about salary payments anyways. So, I really don't think this would add anything to the game that would balance out the confusion it would cause people.
